### ISSUE TITLE
Suggestion: Add a special sign that is only available in Creative Mode.

### ISSUE THREAD

**Issue by MattB70** [Caltinor/DiceMC-Money-Mod#27]
**Suggestion:**
Add a special sign that is only available through Creative Mode or Commands (has no recipe).

**Reason:**
FTBChunks offers the ability to whitelist interactions with specified blocks. Whitelisting all sign interactions would require data generation and would be generally dangerous. If this mod had its own sign item that could be whitelisted, claimed chunks could have shops. Of course my testing isn't completely thorough, so I may have missed an already existing solution to using sign shops in claimed chunks, but this would offer a fairly future proof solution to the issue that hopefully other chunk claiming mods can take advantage of as well.

This definitely isn't completely necessary for my server to function though, just something I noticed when configuring FTBChunks.

**Comment by Caltinor** [comment]
This mod is designed to be client-optional. Adding blocks removes that and makes it client mandatory.

**Comment by MattB70** [comment]
Fair enough, I suspect you could probably achieve the same effect with some nbt on a sign and a custom FTBChunks jar on the server.
